dbt-tests-adapter/dbt/tests/adapter/incremental/test_incremental_microbatch.py

```diff
@@ -63,7 +61,14 @@ def assert_row_count(self, project, relation_name: str, expected_row_count: int)
 
         assert len(result) == expected_row_count, f"{relation_name}:{pformat(result)}"
 
-    @mock.patch.dict(os.environ, {"DBT_EXPERIMENTAL_MICROBATCH": "True"})
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
```

Comment:
Is it worth testing what happens if the flag is turned off?

Reply:
We can do that. We have tests in core I believe that check this, but it makes sense to have a test here as well 🙂